app/src/main/res/values/strings.xml

```diff
@@ -6,4 +6,8 @@
     <string name="main_instruction">The easiest, safest and most convenient way to verify yourself.</string>
     <string name="settings_title">Prepare the test</string>
     <string name="settings_hint">Paste the Token or →</string>
+    <string name="str_hint_email">Email</string>
```

Comment:
we don't use `str_` prefixes, lets keep the names aligned with others? for example `email_hint`, `password_hint`, `login`, `skip_login`?